### DISCUSSION TITLE
<title>Unable to commit and push changes  into main branch

### DISCUSSION THREAD

**Discussion by ajay-marpally** [community/community#45194] [Codespaces]
### Select Topic Area

Product Feedback

### Body

When i click on the button commit it is just taking forever

**Comment by jge162** [comment]
That happened to me today. I first logged out, cleared my cache, closed the browser. Reopened it and commit worked again after. Hopefully it does for you too. But, glad to see I wasn’t the only one.

**Comment by joshspicer** [comment]
Hi @ajay-kanna.  Is this still an issue for you?  If so, could you please provide additional information to help us investigate this.

**Reply by ajay-marpally** [reply]
Hi@joshspicer ,
It is still a issue for me even now..
When I open any of my codespace I can't commit and push them sadly .When I click on commit it just takes forever to commit the changes.

**Reply by dmilburn** [reply]
Hi @ajay-kanna, i'm sorry for the inconvenience. Some additional questions to help us debug:

- have you tried for codespaces in multiple repos or just one?
- have you tried committing in the command line and seeing if that works any differently?
- how do you typically connect to a codespace? in the web? vscode desktop? github CLI?

**Reply by ajay-marpally** [reply]
in multiple rows 
I did nah it doesn't work any different
In web

**Reply by ajay-marpally** [reply]
> in multiple rows I did nah it doesn't work any different In web

@dmilburn
